### PR TITLE
#7 task. andrei-zhivotkov

### PR DESCRIPTION
**Pull request by Inquisitor1229**

### CODE REVIEW

File: task/07-yield-tasks.js [hunk 1]
```diff
@@ -33,7 +33,23 @@
  *
  */
```
Comment (vramaniuk) [inline task/07-yield-tasks.js:34]: Please, provide the link to travis-ci successul build

**Review by vramaniuk — COMMENTED** [review]
add the travis-ci link

File: task/07-yield-tasks.js (hunk 1 above)
Comment (Inquisitor1229) [reply task/07-yield-tasks.js:34]: https://travis-ci.org/github/Inquisitor1229/js-assignments/branches

**Comment by Inquisitor1229** [comment]
Please, check now

**Comment by vramaniuk** [comment]
Ok

**Pull request closed without merging**